### PR TITLE
og image generator

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
## What kind of change does this PR introduce?

Adds og image generation to all sites (currently only docs)

## What is the current behavior?

No og image generator

## What is the new behavior?

Demo of Docs og image.

CleanShot 2022-12-21 at 21 30 06@2x

### CODE REVIEW

**Comment by MildTomato** [comment]
great work @Isaiah-Hamilton 

fonts need updating ?

**Comment by MildTomato** [comment]
@soedirgo @awalias 

We have a supabase function here that we need to sort of keep somewhere in the repo.
Does it have to be in a directory called '/supabase', seems sort of weird.

Also all the toml config stuff - do we need all that?

File: supabase/config.toml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# A string used to distinguish different Supabase projects on the same host. Defaults to the working
```
Comment (MildTomato) [inline supabase/config.toml:1]: @Isaiah-Hamilton maybe we remove this file?
does it even do anything?

File: supabase/functions/og-images/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Open Graph (OG) Image Generator
```
Comment (MildTomato) [inline supabase/functions/og-images/README.md:1]: 🔥

File: supabase/functions/og-images/component/Docs.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          color: 'white',
+          backgroundColor: '#1c1c1c',
+          fontFamily: 'Circular',
+          backgroundImage: 'url(https://obuldanrptloktxcffvn.supabase.co/storage/v1/object/public/images/og-images/gradient.svg)',
+          backgroundRepeat: 'no-repeat',
+        }}
+      >
+        <div tw="flex flex-row items-center justify-center absolute top-[60px] left-[64px]">
+          {icon && (
+            <img src={`https://raw.githubusercontent.com/supabase/supabase/master/apps/docs/public/img/icons/${icon}-icon.svg`} width="50px" height="50px" />
```
Comment (MildTomato) [inline supabase/functions/og-images/component/Docs.tsx:29]: nice idea!

**Review by MildTomato — APPROVED** [review]
approving.
this is getting manually deployed anyway. might as well get it into master

File: supabase/.gitignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Supabase
```
Comment (MildTomato) [inline supabase/.gitignore:1]: might also not need this

Comment (Isaiah-Hamilton) [reply supabase/.gitignore:1]: this file would be good to keep

**Comment by saltcod** [comment]
Awesome @Isaiah-Hamilton!   Great work!

**Pull request merged**